docs/articles/documentation/getting-started/index.md

```diff
@@ -0,0 +1,154 @@
+You declare fixtures by using the `fixture` function.
+You can optionally specify a start page URL for a fixture by using the [page](../test-api/test-code-structure.md#specifying-the-start-webpage) function.
+In this tutorial, you will create a test for the [http://testcafe.devexpress.com/example](http://testcafe.devexpress.com/example) sample page.
+
+Test code is placed within the `test` function.
+
+```js
+fixture `Getting Started`;
+    .page('http://testcafe.devexpress.com/example');
+
+test('MyTest', async t => {
+    //Test code
+});
+```
+
+### Performing Actions on Page
+
+Every test should be capable of interacting with a page content. To perform user actions, TestCafe provides a number of [actions](../test-api/actions.md): `click`, `hover`, `typeText`, `upload`, etc.
+They can be called in a chain.
+
+The following fixture contains a simple test that types a developer name into a text editor and then clicks the Submit button.
+You can try running this test against the sample page by yourself.
+
+```js
+fixture `Getting Started`
+    .page('http://testcafe.devexpress.com/example');
+
+test('MyTest', async t => {
+    await t
+        .typeText('#developer-name', 'John Smith')
+        .click('#submit-button');
+});
+```
+
+All test actions are implemented as asynchronous functions of the test controller object `t`. This object is used to access test run API.
+To wait for actions to be complete, use the `await` function when calling these actions or action chains.
+
+### Observing Page State
+
+TestCafe allows you to observe the state of page elements.
+For this purpose, it provides the `Hybrid` function.
+You just write your code within this function, and TestCafe executes it on the client side, thus giving you direct access to DOM elements.
+
+The following code demonstrates how to use a parameterized hybrid function to obtain text content of an element identified by the `id` attribute.
+
+```js
+import { Hybrid } from 'testcafe';
+
+const getElementTextById = Hybrid((id) => document.querySelector(`#${id}`).textContent);
+```
+
+For more information, see [Executing Client Code](../test-api/executing-client-code.md).
+
+### Assertions
+
+A functional test should check the result of actions performed.
+For example, clicking the Submit button on the sample web page opens a "Thank you" page whose header addresses a user by the entered name.
+To check if the name is correct, you have to add an assertion to the test.
+
+You can use assertions from Node's built-in [assert](https://nodejs.org/api/assert.html) module or from any third-party assertion library.
+Before calling assertions, make sure an assertion library is installed into your tested project.
+
+The following test demonstrates how to use an assertion from [Chai Assertion Library](http://chaijs.com/api/bdd/).
+Before running the test, install the assertion library: go to the tested project directory (*my-project*) and run `npm install chai`.
```

Comment:
Skip `go to the tested project directory (*my-project*)` part. 
It should be `npm install --save-dev chai`